client/src/components/leftSidebar/types.ts

```diff
@@ -0,0 +1,11 @@
+import { RootState } from "../../reducers";
+
+export interface Props {
+  scatterplotXXaccessor: string;
+  scatterplotYYaccessor: string;
```

Comment:
Let's grab these straight from the store! YMMV on my suggestion
```suggestion
  scatterplotXXaccessor: RootState["controls"]["scatterployXXaccessor"];
  scatterplotYYaccessor:  RootState["controls"]["scatterployYYaccessor"];
```